src/modules/storage/LedgerStorage.ts

```diff
@@ -4111,7 +4128,7 @@ export class LedgerStorage extends Storages {
                     blocks B
                     INNER JOIN transactions T ON (B.height = T.block_height)
                     INNER JOIN tx_inputs I ON (T.tx_hash = I.tx_hash)
-                    INNER JOIN tx_outputs S ON (I.utxo = S.utxo_key)
+                    INNER JOIN tx_outputs S ON (I.utxo = S.utxo_key and I.tx_index = S.tx_index)
```

Comment:
Because address was replicating, if one address has more than one transaction in one block, address's utxo will match on every transaction, that why I've added tx_index